src/patternfly/components/DualListSelector/dual-list-selector-status-text.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-dual-list-selector__status-text{{#if dual-list-selector-status-text--modifier}} {{dual-list-selector-status-text--modifier}}{{/if}}"
```

Comment:
```suggestion
<span class="pf-c-dual-list-selector__status-text{{#if dual-list-selector-status-text--modifier}} {{dual-list-selector-status-text--modifier}}{{/if}}"
```

Reply:
I added this text element anticipating there might be icons, buttons, etc that get added here. Do you think this should be a span, or should the `__status` parent be a flex layout, so we can easily place things before/after the text?

Reply:
I would suggest both `__status { display: flex }` to support things before/after and `span.pf-c-dual-list-selector__status-text` to be consistent with other `__text` elements. Although, we could update `__status { display: flex }` when that becomes necessary.

Reply:
I'll go ahead and add `flex` since doing that after the fact is a breaking change since it will change the basic layout of all children of the `__status` element.